libraries/AC_AttitudeControl/AC_AttitudeControl_Multi_6DoF.h

```diff
@@ -19,9 +19,9 @@ class AC_AttitudeControl_Multi_6DoF : public AC_AttitudeControl_Multi {
     }
 
     // Command a Quaternion attitude with feedforward and smoothing
-    // attitude_desired_quat: is updated on each time_step (_dt) by the integral of the angular velocity
+    // attitude_desired_quat: is updated on each time_step (_dt) by the integral of the body frame angular velocity
     // not used anywhere in current code, panic so this implementaiton is not overlooked
-    void input_quaternion(Quaternion& attitude_desired_quat, Vector3f ang_vel_target) override;
+    void input_quaternion(Quaternion& attitude_desired_quat, Vector3f ang_vel_body) override;
```

Comment:
should be:
  const Vector3f &ang_vel_body

Reply:
Done

Reply:
Changed it and realised that the input quaternion modifies the parameter on the stack. Happy to fix this on all the code I am responsible for in a single PR.